requirements/runtime.txt

```diff
@@ -16,3 +16,4 @@ onnxoptimizer
 onnxruntime>=1.1.2
 Shapely==1.8.0
 editdistance
+nncf @ git+https://github.com/openvinotoolkit/nncf@464244204fc2c5e80c8164c17d8d266ccae50062#egg=nncf
```

Comment:
Is there any chance to use release version of NNCF at some point?

Reply:
yes, we will use release version in SC1.1